cmd/soroban-cli/src/commands/snapshot/create.rs

```diff
@@ -26,17 +26,18 @@ use stellar_xdr::curr::{
 use tokio::fs::OpenOptions;
 
 use crate::{
-    commands::{config::data, HEADING_RPC},
+    commands::{config::data, global, HEADING_RPC},
     config::{self, locator, network::passphrase},
+    output,
     utils::{get_name_from_stellar_asset_contract_storage, parsing::parse_asset},
 };
 
 #[derive(Clone, Copy, Debug, Eq, Hash, PartialEq, ValueEnum)]
-pub enum Output {
+pub enum OutputFile {
     Json,
 }
 
-impl Default for Output {
+impl Default for OutputFile {
```

Comment:
We use the term Output elsewhere for the output type, where the output isn't a file, could we keep it as is, and change the `output` module to `print` since it's for printing specifically?